tutorials/FSS.ipynb

```diff
@@ -0,0 +1,2354 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ee0d9b1c-9e0a-4fe1-891a-a74450c1dd3b",
+   "metadata": {},
+   "source": [
+    "# Fractions Skill Score (FSS)\n",
+    "\n",
+    "The original paper, and formal definitions can be found here:  \n",
+    "https://journals.ametsoc.org/view/journals/mwre/136/1/2007mwr2123.1.xml\n",
+    "\n",
+    "For an explanation of the FSS, and implementation considerations, see:  \n",
+    "https://www.researchgate.net/publication/269222763_Fast_calculation_of_the_Fractions_Skill_Score"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2162e15f-0bbc-4b5a-856d-f70d550c779e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## FSS for a single 2D field\n",
+    "FSS is computed over 2D arrays representing the observations & forecasts in the spatial domain. The user has to make sure that the input dimensions correspond to the spatial domain e.g. `lat x lon`. Generally the computation involves sliding a window over the input field(s) and applying a threshold over the fcst and obs values.\n",
+    "\n",
+    "The resulting binary field is summed up to represent the populace (number of ones/\"true\" values in the window).\n",
+    "\n",
+    "The resulting 2-D field of rolling sums represents [\"_Integral Image_\"][integral_image] of the respective forecast and obeservation fields, which is then aggregated over all the sliding windows to compute the fractions skill score.\n",
+    "\n",
+    "The FSS is then roughly defined as:\n",
+    "$$\n",
+    "    fss = 1 - \\frac{\\sum_{w}(p_o - p_f)^2}{\\sum_{w}p_o^2 + \\sum_{w}p_f^2}\n",
+    "$$\n",
+    "\n",
+    "where,\n",
+    "- $p_o$ is the observation populace > threshold, in one window\n",
+    "- $p_f$ is the forecast populace > threshold, in one window\n",
+    "- $\\sum_{w}$ is the sum over all windows\n",
+    "\n",
+    "The implementation details are beyond the scope of this tutorial please refer to, [Fast calculation of the Fractions Skill Score][fss_ref] for more info\n",
+    "\n",
+    "In summary, computation of a single field requires the following parameters:\n",
+    "- forecast 2-D field (in spatial domain)\n",
```

Comment:
Consider adding an empty line before the dot points, so the dot points render properly in readthedocs.